services/README.md

```diff
@@ -47,11 +47,11 @@ azure_mgmt_storage_2018_02 = { package = "azure_mgmt_storage", git = "https://gi
 ```
 
 ## Data Plane Crates
-The control plane crates will be named `azure_svc_${specification_directory}`, such as `azure_svc_storage`. 
+The data plane crates will be named `azure_svc_${specification_directory}`, such as `azure_svc_storage`. 
```

Comment:
The data plane crates should ideally match our naming convention for other SDKs. Some like .NET and Java use possible topic-level names like Azure.Security.KeyVault.Secrets while languages like Python and JS simplify it, which may be more applicable here e.g. @azure/keyvault-secrets or azure_keyvault_secrets.

Control (or management) planes are typically Azure.ResourceManager.KeyVault.

Reply:
/cc @rylev @MindFlavor